ddtrace/context.py

```diff
@@ -29,6 +32,8 @@ class Context(object):
         "_lock",
         "_meta",
         "_metrics",
+        "_upstream_services",
```

Comment:
idk if we want to update the `repr` for these new attributes or not

Reply:
it will show up in meta

Reply:
remember, these two properties are to maintain the state of "my inherited upstream service entr(y|ies)" and "my upstream service entry", but the current active/correct/propagatable (this should be a word) concatenation of these lives in `_meta`